annlite/container.py

```diff
@@ -300,6 +300,8 @@ def update(
         cells: 'np.ndarray',
         docs: 'DocumentArray',
     ):
+        update_success = 0
```

Comment:
what about adding a new parameter, such as `raise_errors_on_not_found: bool = True`. At the executor endpoint, setting `raise_errors_on_not_found=False`